os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_app_main.c

```diff
@@ -52,7 +52,7 @@
  *                              Variables
  *============================================================================*/
 void print_no_server(const char* format, ...) {}
-
+#if 0

```

Comment:
Please remove commented variables if they are not used anymore.

Reply:
Done.